vue/components/ui/organisms/header/assets/apps/registry.svg

```diff
@@ -0,0 +1 @@
+<svg width="120" height="120" viewBox="0 0 120 120" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient x1="48.747%" y1="48.771%" x2="49.746%" y2="48.806%" id="a"><stop stop-color="#6051F2" offset="0%"/><stop stop-color="#645AF3" offset="10%"/><stop stop-color="#7275F6" offset="45%"/><stop stop-color="#7A86F8" offset="76%"/><stop stop-color="#7D8CF9" offset="100%"/></linearGradient><linearGradient x1=".018%" y1="50.009%" x2="99.918%" y2="50.009%" id="b"><stop stop-color="#6051F2" offset="0%"/><stop stop-color="#7D8CF9" offset="100%"/></linearGradient></defs><g fill="none"><path fill="url(#a)" opacity=".47" d="M110.508 117.108l-.02-.02.02.02z"/><path d="M75.508 113.852V85.865a5.538 5.538 0 015.538-5.447h27.877a5.538 5.538 0 015.539 5.447v27.987a5.538 5.538 0 01-5.539 5.428h-27.97a5.538 5.538 0 01-5.445-5.428zm38.954-51.083V34.763a5.538 5.538 0 00-5.539-5.428h-27.97a5.538 5.538 0 00-5.538 5.428V62.77a5.538 5.538 0 005.539 5.446h27.97a5.538 5.538 0 005.538-5.446zm-50.437 50.345V85.126a5.538 5.538 0 00-5.447-5.538H30.591a5.538 5.538 0 00-5.428 5.538v27.988a5.428 5.428 0 005.428 5.428h27.987a5.538 5.538 0 005.447-5.428zm-48-76.394L4.32 44.308a5.428 5.428 0 00-1.588 7.569l7.625 11.76a5.538 5.538 0 007.532 1.588L29.631 57.6a5.538 5.538 0 001.606-7.532l-7.643-11.742a5.538 5.538 0 00-7.514-1.606h-.055zm40.08 20.862l9.23-10.524a5.538 5.538 0 00-.48-7.68l-10.504-9.23a5.538 5.538 0 00-7.68.48L37.44 41.132a5.538 5.538 0 00.48 7.68l10.505 9.231a5.538 5.538 0 007.735-.461h-.055zM14.492 15.637l7.385 11.926a5.538 5.538 0 007.495 1.846l11.926-7.384a5.538 5.538 0 001.847-7.496L35.76 2.603A5.538 5.538 0 0028.265.757L16.338 8.142a5.428 5.428 0 00-1.846 7.495z" transform="translate(3)" fill="url(#b)"/></g></svg>
```

Comment:
`npm run pretty`

Reply:
I had ran `pretty`, it sets the svg like that